<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>动力系统演示平台</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>761</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>控制面板</string>
       </property>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>351</width>
          <height>61</height>
         </rect>
        </property>
        <property name="title">
         <string>仿真系统</string>
        </property>
        <widget class="QComboBox" name="systemSelect">
         <property name="geometry">
          <rect>
           <x>17</x>
           <y>22</y>
           <width>171</width>
           <height>22</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>351</width>
          <height>371</height>
         </rect>
        </property>
        <property name="title">
         <string>系统方程</string>
        </property>
        <widget class="QLabel" name="funLabel">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>40</y>
           <width>291</width>
           <height>291</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Times New Roman</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>参数控制</string>
       </property>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>361</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>时间</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>开始时间</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>188</x>
           <y>30</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>结束时间</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>时间步长</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit">
         <property name="geometry">
          <rect>
           <x>66</x>
           <y>26</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>25</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_3">
         <property name="geometry">
          <rect>
           <x>66</x>
           <y>60</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>361</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>初始值</string>
        </property>
        <widget class="QLabel" name="var1">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>30</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>var1</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_4">
         <property name="geometry">
          <rect>
           <x>56</x>
           <y>26</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="var1_2">
         <property name="geometry">
          <rect>
           <x>180</x>
           <y>30</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>var1</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_5">
         <property name="geometry">
          <rect>
           <x>216</x>
           <y>26</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="var1_3">
         <property name="geometry">
          <rect>
           <x>19</x>
           <y>64</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>var1</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_6">
         <property name="geometry">
          <rect>
           <x>55</x>
           <y>60</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
        <widget class="QLabel" name="var1_4">
         <property name="geometry">
          <rect>
           <x>181</x>
           <y>60</y>
           <width>54</width>
           <height>12</height>
          </rect>
         </property>
         <property name="text">
          <string>var1</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_7">
         <property name="geometry">
          <rect>
           <x>217</x>
           <y>56</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>351</width>
          <height>241</height>
         </rect>
        </property>
        <property name="title">
         <string>仿真控制</string>
        </property>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>331</width>
           <height>211</height>
          </rect>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QPushButton" name="dataExport">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>30</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>数据导出</string>
          </property>
         </widget>
         <widget class="QPushButton" name="start">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>开 始</string>
          </property>
         </widget>
         <widget class="QPushButton" name="visual">
          <property name="geometry">
           <rect>
            <x>230</x>
            <y>30</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>作  图</string>
          </property>
         </widget>
         <widget class="QGroupBox" name="groupBox_9">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>301</width>
            <height>81</height>
           </rect>
          </property>
          <property name="title">
           <string>作图控制</string>
          </property>
          <widget class="QRadioButton" name="radioButton1">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>41</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>3D</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton2">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>20</y>
             <width>41</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>2D</string>
           </property>
          </widget>
          <widget class="QRadioButton" name="radioButton3">
           <property name="geometry">
            <rect>
             <x>140</x>
             <y>20</y>
             <width>71</width>
             <height>16</height>
            </rect>
           </property>
           <property name="text">
            <string>时间序列</string>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox1">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>50</y>
             <width>51</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QComboBox" name="comboBox2">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>50</y>
             <width>51</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>170</y>
            <width>81</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>SimSun-ExtB</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>530</y>
      <width>501</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>SimSun-ExtB</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>动力系统仿真平台 西北工业大学理学院</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
